storefront-bff/src/main/java/com/yas/storefrontbff/config/SecurityConfig.java

```diff
@@ -28,6 +28,7 @@ public SecurityWebFilterChain springSecurityFilterChain(ServerHttpSecurity http)
         http
                 .authorizeExchange()
                 .pathMatchers("/profile/**").authenticated()
+                .pathMatchers("/address/**").authenticated()
```

Comment:
Just my opinion, but I think that since "profile" and "address" have already been verified by "customer service," should  we remove both of it?